app/Providers/AppServiceProvider.php

```diff
@@ -1,10 +1,12 @@
 <?php
 
 namespace App\Providers;
-
 use Illuminate\Support\ServiceProvider;
 
 class AppServiceProvider extends ServiceProvider
+
+
+
 {
```

Comment:
Opening brace of a class must be on the line following the class declaration; found 3 line(s)